apps/test/unit/p5lab/spritelab/SpritelabTest.js

```diff
@@ -145,6 +146,32 @@ describe('SpriteLab', () => {
         expect(resultingAnimations.orderedKeys.length).to.be.equal(1);
       });
 
+      describe('react to executionError', () => {
+        let alertSpy;
+        beforeEach(() => {
+          alertSpy = sinon.stub(studioApp(), 'displayWorkspaceAlert');
+          sinon.stub(instance, 'getMsg').returns({
+            workspaceAlertError: () => {
+              return 'translated string';
+            }
+          });
+        });
+        afterEach(() => {
```

Comment:
style nit: add newlines between expressions for readability. the structure i usually try to enforce is

```javascript
describe('', () => {
  beforeEach(() => {});

  afterEach(() => {});

  it('', () => {});

  it('', () => {});
});
```